Bottom: b3d5a2afe071cd15b927b29ec845455143ca14b0
Top:    62cfc1e0a4045e7ed7e1af8024ba3a09e54c00ef
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-26 23:26:06 +0100

Refresh of patch

---

diff --git a/src/const.h b/src/const.h
index 6fa7cbc..193ebc8 100644
--- a/src/const.h
+++ b/src/const.h
@@ -36,13 +36,20 @@
 #include "globalconst.h"
 
 /* ---------------------------------------------------------------------------
+ * integer codings for the board sides.
+ */
+#define BOTTOM_SIDE             0
+#define TOP_SIDE                1
+
+
+/* ---------------------------------------------------------------------------
  * the layer-numbers of the two additional special layers
  * 'top' and 'bottom'. The offset of MAX_LAYER is not added
  */
-#define	BOTTOM_SILK_LAYER	0
-#define	TOP_SILK_LAYER		1
-#define	BOTTOM_SOLDERMASK_LAYER	2
-#define	TOP_SOLDERMASK_LAYER	3
+#define BOTTOM_SILK_LAYER       0
+#define TOP_SILK_LAYER          1
+#define BOTTOM_SOLDERMASK_LAYER 2
+#define TOP_SOLDERMASK_LAYER    3
 
 /* ---------------------------------------------------------------------------
  * misc constants
